app/src/main/java/com/dgaf/happyhour/Controller/DealListFragment.java

```diff
@@ -70,6 +80,18 @@ public void onActivityCreated(Bundle savedInstanceState) {
 
         mRecyclerView.addItemDecoration(new DealListDecoration(getActivity()));
 
+        onRefresh();
+
+    }
+
```

Comment:
You want to expand on this one a bit more? I don't quite understand what you mean.